MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -194,20 +194,18 @@ - (void) handleRenditionChange:(NSNotification *) notif {
 - (void) handleRenditionChangeInAccessLog:(AVPlayerItemAccessLog *) log {
     AVPlayerItemAccessLogEvent *lastEvent = log.events.lastObject;
     float advertisedBitrate = lastEvent.indicatedBitrate;
-    BOOL bitrateHasChanged = ![self doubleValueIsEqual:@(_lastAdvertisedBitrate) toOther:@(advertisedBitrate)];
-    BOOL isStartingPlayback = [self doubleValueIsEqual:@(_lastAdvertisedBitrate) toOther:@(0)];
-
-    if (bitrateHasChanged) {
-        if(isStartingPlayback) {
-            // This is not a renditionchange but the player playing the first rendition.
-            _lastAdvertisedBitrate = advertisedBitrate;
-            return;
-        }
-        NSLog(@"MUXSDK-INFO - Switch advertised bitrate from: %f to: %f", _lastAdvertisedBitrate, advertisedBitrate);
-        [[NSNotificationCenter defaultCenter] postNotificationName:RenditionChangeNotification object: @{
-            RenditionChangeNotificationInfoAdvertisedBitrate: @(advertisedBitrate)
-        }];
+    if (advertisedBitrate == 0 && !_started) {
```

Comment:
this condition should be different than 0, otherwise it means it would only assign this first _lastAdvertisedBitrate = 0.